tests/install_testbed/install_neutron_controller.sh

```diff
@@ -0,0 +1,91 @@
+./openstack-config --set $file DEFAULT rabbit_password $RABBIT_PASS
+./openstack-config --set $file DEFAULT auth_strategy keystone
+./openstack-config --set $file keystone_authtoken auth_uri $AUTH_URI
+./openstack-config --set $file keystone_authtoken identity_uri $IDENTITY_URI
+./openstack-config --set $file keystone_authtoken admin_tenant_name service
+./openstack-config --set $file keystone_authtoken admin_user neutron
+./openstack-config --set $file keystone_authtoken admin_password $NEUTRON_PASS
+./openstack-config --set $file DEFAULT core_plugin ml2
+./openstack-config --set $file DEFAULT service_plugins router
+./openstack-config --set $file DEFAULT allow_overlapping_ips True
+./openstack-config --set $file DEFAULT notify_nova_on_port_status_changes True
+./openstack-config --set $file DEFAULT notify_nova_on_port_data_changes True
+./openstack-config --set $file DEFAULT nova_url http://$CONTROLLER:8774/v2
+./openstack-config --set $file DEFAULT nova_admin_auth_url ${IDENTITY_URI}/v2.0
+./openstack-config --set $file DEFAULT nova_region_name $REGION
+./openstack-config --set $file DEFAULT nova_admin_username nova
+./openstack-config --set $file DEFAULT nova_admin_tenant_id $SERVICE_TENANT_ID
+./openstack-config --set $file DEFAULT nova_admin_password $NOVA_PASS
+
+file=/etc/neutron/plugins/ml2/ml2_conf.ini
+./openstack-config --set $file ml2 type_drivers flat,gre
+./openstack-config --set $file ml2 tenant_network_types gre
+./openstack-config --set $file ml2 mechanism_drivers openvswitch
+./openstack-config --set $file ml2_type_gre tunnel_id_ranges 1001:2000
+./openstack-config --set $file securitygroup enable_security_group True
+./openstack-config --set $file securitygroup enable_ipset True
+./openstack-config --set $file securitygroup firewall_driver neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver
+
+
+file=/etc/nova/nova.conf
+./openstack-config --set $file DEFAULT network_api_class nova.network.neutronv2.api.API
+./openstack-config --set $file DEFAULT security_group_api neutron
+./openstack-config --set $file DEFAULT linuxnet_interface_driver nova.network.linux_net.LinuxOVSInterfaceDriver
+./openstack-config --set $file DEFAULT firewall_driver nova.virt.firewall.NoopFirewallDriver
+./openstack-config --set $file neutron url http://$CONTROLLER:9696
+./openstack-config --set $file neutron auth_strategy keystone
+./openstack-config --set $file neutron admin_auth_url ${IDENTITY_URI}/v2.0
+./openstack-config --set $file neutron admin_tenant_name service
+./openstack-config --set $file neutron admin_username neutron
+./openstack-config --set $file neutron admin_password $NEUTRON_PASS
+./openstack-config --set $file neutron service_metadata_proxy True
+./openstack-config --set $file neutron metadata_proxy_shared_secret $METADATA_SECRET
+
+##### populate the database
+su -s /bin/sh -c "neutron-db-manage --config-file /etc/neutron/neutron.conf \
+  --config-file /etc/neutron/plugins/ml2/ml2_conf.ini upgrade juno" neutron
```

Comment:
Yes, these scripts are for the Juno version. It is the most recent version we have installation scripts. There are also some scripts to upgrade from Juno to Kilo, but probably it is easiest  now that we have a working version that was the first objective, upgrading the scripts.